src/MentionsInputLayout.ts

```
@@ -48,6 +60,111 @@ export interface ViewSyncDecision {
   flushNow: boolean
 }
 
+const getRecordKeys = (record: Record<number, unknown>): string[] => Object.keys(record).sort()
```

ESLint

[error] Use `Array#toSorted()` instead of `Array#sort()`.

(unicorn/no-array-sort)